tests/server/helpers.js

```diff
@@ -13,11 +13,12 @@ export function checkSRI(res) {
   const $script = $('script[src]');
   assert.ok($script.length > 0, 'must be at least 1 script');
   $script.each((i, elem) => {
-    if (!$(elem).attr('src').includes('analytics.js')) {
-      assert.include($(elem).attr('integrity'),
-        'sha512', 'script should have integrity attr');
-      assert.equal($(elem).attr('crossorigin'),
-        'anonymous', 'script should have crossorigin attr');
+    if ($(elem).attr('src').includes('analytics.js')) {
```

Comment:
Might be nice to do `const $elem = $(elem);` at the top of this loop, looks like we're doing that work three times.